augur/data/schema-export-v2.json

```diff
@@ -130,7 +130,7 @@
             "type" : "object",
             "$comment": "The phylogeny in a nested JSON structure",
             "additionalProperties": false,
-            "required": ["name"],
+            "required": ["name", "node_attrs"],
```

Comment:
Will this change and the two below to required attributes require a major version release for Augur? I can't remember how we decided to treat schema updates, but since more external software now consumes Auspice JSONs, we might want to announce these changes more loudly/broadly.

Reply:
In the context of Auspice a tree without `node_attrs` was never valid, in the sense that it wouldn't render correctly (because this implies no distance metric). In the context of Augur perhaps such datasets existed? I can't think of any situations where it would be used, but maybe?

Reply:
Would be good to add to change log, but probably no major version needed?

Reply:
Added changelog entry mentioning this